src/main/java/net/sf/jabref/logic/xmp/XMPUtil.java

```diff
@@ -147,7 +136,7 @@ public static void writeXMP(String filename, BibEntry entry,
 
             if (meta.isPresent()) {
 
-                List<XMPSchema> schemas = meta.get().getSchemasByNamespaceURI(XMPSchemaBibtex.NAMESPACE);
+                List<XMPSchema> schemas = meta.get().getAllSchemas();
```

Comment:
I also tried around with pdfbox and came to that solution:
      XMPSchemaBibtex bib =  (XMPSchemaBibtex) meta.get().getSchema(XMPSchemaBibtex.class);